Mathlib/Analysis/Normed/Field/Ultra.lean

```diff
@@ -0,0 +1,63 @@
+/-
+Copyright (c) 2024 Yakov Pechersky. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yakov Pechersky
+-/
+import Mathlib.Analysis.Normed.Field.Basic
+import Mathlib.Analysis.Normed.Group.Ultra
+
+/-!
+# Ultrametric norms on division rings
+
+This file contains results on the behavior of norms in ultrametric division rings.
+
+## Main results
+
+* `norm_intCast_le_one`:
+  the norm of the image of an integer in the division ring is always less than or equal to one
+
+## Tags
+
+ultrametric, nonarchimedean
+-/
+open Metric NNReal
+
+namespace IsUltrametricDist
+
+section DivisionRing
+
+variable {K : Type*} [NormedDivisionRing K] [IsUltrametricDist K]
+
+lemma norm_add_one_le_max_norm_one (x : K) :
+    ‖x + 1‖ ≤ max ‖x‖ 1 := by
+  simpa using norm_add_le_max x 1
+
+lemma nnnorm_add_one_le_max_nnnorm_one (x : K) :
+    ‖x + 1‖₊ ≤ max ‖x‖₊ 1 := by
+  simpa using norm_add_le_max x 1
```

Comment:
`simpa using` isn't using anything here.

Reply:
How so? It needs to call `(nn)norm_one`.

Reply:
I'll use defeq with the norm lemma.